### DISCUSSION TITLE
How about generating and filtering a full list of all installed packages?

### DISCUSSION THREAD

**Discussion by BingoKingo** [topgrade-rs/topgrade#509] [Ideas]
Topgrade is a easy upgrading tool,  could it list all packages installed by different packages in one command? Will it support packages list backup and restore?

**Comment by SteveLauC** [comment]
> could it list all packages installed by different packages in one command?

AFAIK, there is no such command in Topgrade.

> Will it support packages list backup and restore?

Currently, Topgrade ONLY supports updating packages, achieving backup and restore would need extra huge work. There are some [discussions](https://github.com/topgrade-rs/topgrade/discussions/31#discussion-4482507) about working on something you requested, but I am afraid it won't land in the near future
